lib/binaries/chrome_driver.ts

```diff
@@ -30,7 +30,13 @@ export class ChromeDriver extends Binary {
 
   suffix(ostype: string, arch: string): string {
     if (ostype === 'Darwin') {
-      return 'mac32' + this.suffixDefault;
+      // after chromedriver version 2.23, the name of the binary changed
+      // They no longer provide a 32 bit binary
+      if (parseFloat(this.version()) >= 2.23) {
```

Comment:
Please don't do this with floats. Float inaccuracies will likely affect `parseFloat` the same way as the literals. (But why take the risk?) But still `2.3` will compare greater than `2.23`, while `2.100` will compare as less.